src/server/generic_family.cc

```diff
@@ -724,6 +747,23 @@ OpResult<void> GenericFamily::OpRen(const OpArgs& op_args, string_view from_key,
   return OpStatus::OK;
 }
 
+OpResult<uint32_t> GenericFamily::OpStick(const OpArgs& op_args, ArgSlice keys) {
+  DVLOG(1) << "Stick: " << keys[0];
+
+  auto& db_slice = op_args.shard->db_slice();
+
+  uint32_t res = 0;
+  for (uint32_t i = 0; i < keys.size(); ++i) {
+    auto [it, _] = db_slice.FindExt(op_args.db_ind, keys[i]);
+    if (IsValid(it) && !it->second.IsSticky()) {
+      it->second.SetSticky(true);
```

Comment:
I wonder if we should set stickiness on a key or on a value? see my questions below

Reply:
I first assumed that the key is more suitable for this job, but then I've discovered that expiration is marked on the value. The questions work for both the key & value (I've added tests).

On a second thought, expiration is marked on the value to persist a `RENAME`. Stickyness should be bound strictly to keys? So I'll probably switch to the key?

Reply:
yes, expiration is marked for the value, because of the `set` operation that destructs the expiry property. append or incremental command do not though.  I guess I could still put it on a key and handle the `set` edge-case. Maybe it would even end up simpler.... 

anyway, in case of STICK, I do think it makes sense to destruct this property upon writes (besides DEL of course), so `key` seems like a sane choice to me. 

I did not think of RENAME before. I wonder if it's even a documented behavior for expiration. If expiration persists a rename, STICK should also persist it.